### ISSUE TITLE
Docs: explain off-site vs. OCF hosting

### ISSUE THREAD

**Issue by matthew-mcallister** [ocf/ocfweb#56]
A common question, brought up in [rt#3973](https://rt.ocf.berkeley.edu/Ticket/Display.html?id=3973). Should probably distinguish free vs. proprietary software, off-site vs. on-site, web software versus web hosting service (and how the latter two are often intentionally muddled up). Also give a somewhat more complete list of available software.

**Comment by hexecute** [comment]
> free vs. proprietary software  

Do we really need to do that here? I think that might just make people angry without any benefits (e.g. "We can't use X because of OCF's political views?"). Instead, we could approach it as a requirement of the university for security purposes which is more accurate (I think?).

**Comment by kpengboy** [comment]
Or cut to the chase and explain that it is not physically possible for us
to host Wix etc. sites?

**Comment by matthew-mcallister** [comment]
I definitely don't recommend posting an entire Stallman article. It should be clear why we cannot host a Wix website. Example:

> Many "website builder" style services, such as Wix, Weebly, use a proprietary software backend owned by their companies. The OCF doesn't have the right or ability to transfer any website made with those services to our servers, so we cannot host them on-site. On the other hand, we can run anything using free and open-source software, like WordPress and Rails, so make your website with those!

**Comment by nattofriends** [comment]
No. This is not about free software. Make it abundantly clear that what Wix
asks people to do is not OCF hosting a site, it is OCF pointing signs to
Wix hosting the site themselves. It is due to University rules that we
cannot do this.

Then proceed to describe the possible solutions, such as submitting an
offsite request, or using free software hosted on the OCF.

**Comment by matthew-mcallister** [comment]
I agree, which is why I didn't say anything about the merits/demerits of free/proprietary software. However, a user might rightly wonder "Why isn't there a way to copy my Wix website over to the OCF servers?" After all, you can move from WordPress.com to WordPress on the OCF.

It sounds like you're referring to some specific user support page on Wix which tells people to ask for DNS to point to Wix. Yes, we should explicitly warn people that we cannot do that. If it needs to say more explicitly "No, we cannot give you offsite DNS *nor* can we copy your Wix website to our servers", then so be it. But it deserves at least a one-sentence explanation.

**Comment by nattofriends** [comment]
Sure, that makes sense. I will add that Wordpress and Tumblr also have
CNAME handling support, so it isn't limited to just Wix.

2015-10-05 20:19 GMT-07:00 Matthew McAllister <notifications@github.com>:
